iina/VideoView.swift

```diff
@@ -310,12 +310,16 @@ class VideoView: NSView {
       }
     }
 
-    if videoLayer.colorspace != VideoView.deviceRGBColorspace {
-      Logger.log("Returning to deviceRGB color space", subsystem: hdrSubsystem)
+    if videoLayer.colorspace != VideoView.SRGB {
+      videoLayer.colorspace = VideoView.SRGB;
       videoLayer.wantsExtendedDynamicRangeContent = false
-      videoLayer.colorspace = VideoView.deviceRGBColorspace
       player.mpv.setString(MPVOption.GPURendererOptions.targetTrc, "auto")
       player.mpv.setString(MPVOption.GPURendererOptions.targetPrim, "auto")
+      player.mpv.setString(MPVOption.GPURendererOptions.targetPeak, "auto")
+      player.mpv.setString(MPVOption.GPURendererOptions.toneMapping, "auto")
+      player.mpv.setString(MPVOption.GPURendererOptions.toneMappingParam, "default")
+      player.mpv.setString(MPVOption.GPURendererOptions.toneMapping, "")
```

Comment:
Why did you set this option twice? On line 319
```
player.mpv.setString(MPVOption.GPURendererOptions.toneMapping, "auto")
```

Reply:
Because it is that way in the code in the other repository and I failed to notice that when preparing the merge. GOOD CATCH!

The "auto" for toneMapping was a recent add by mpv, so maybe the empty string setting was supposed to be removed after upgrading to 035.1?

@CarterLi Which is the correct value to use?

I'm going to guess "auto" and remove the setting to the empty string.

Reply:
Just use the default value of MPV

Reply:
The default is "auto", so I removed this [line](https://github.com/CarterLi/iina/blob/a7e5cb961b8f4b209d3532e8653ef73f0d88415c/iina/VideoView.swift#L292) that is in IINA+